README.md

```diff
@@ -181,6 +182,24 @@ runSqlExample().catch((error) => {
 
 ```
 
+### setLibraryList
+
+Change to system naming and set the library list (using `CHGLIBL`) of the connection.
+
+```javascript
+async function execExample() {
+  const connection = new Connection({ url: '*LOCAL' });
+
+  await connection.setLibraryList(['QIWS']);
```

Comment:
Might be better to show an example with an array of multiple libraries just to be explicit here

Reply:
Added `await connection.setLibraryList(['QIWS', 'QXMLSERV']);`